### ISSUE TITLE
MSBuild task invalid outdir with multiple projects

### ISSUE THREAD

**Issue by AndersSahlin** [microsoft/TypeScript#7806]
Using Typescript compiler Nuget 1.8.6 (`Microsoft.TypeScript.MSBuild`). 

I have created an example solution with two projects, a Web project (TsExampleProject) that contains `A.ts` and a class library (TsLib) that contains `B.ts`. The web project is referencing the class library via a definition file.

The Typescript output is configured to end up in `output\ts`, and since multiple projects are referenced the compiler places them in a folder structure like this: `\output\ts\TsExampleProject\A.js` and `output\ts\TsLib\B.js`.

When you try to build this solution using MsBuild (`build.bat` file in the root) it fails with the following error message: 

>  C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\WebApplications\Microsoft.WebApplication.targets(182,5): error MSB3030: Could not copy the file "output\ts\A.js.map" because it was not found. [C:[...]\TypescriptIssueRepo2\TsExampleProject\TsExampleProject.csproj]

So it seems like the MsBuild task is not looking for the output files in the correct folder, with regards to the multiple project structure.

Example repo: https://github.com/AndersSahlin/TypescriptIssueRepo2/

**Comment by mhegazy** [comment]
this is the same issue as https://github.com/Microsoft/TypeScript/issues/6934. The compiler settings in the tsconfig are not passed along to other steps in the build pipeline, e.g. publish.

**Comment by paulvanbrenk** [comment]
Ok, I looked at this issue again and it's a different issue from the one we duped this against. 

Is this actually a regression from a previous release? Since the Web Project doesn't have any knowledge of the .ts files in the Library Project, and the output .js files, I don't think this worked  The reason being that TypeScript project don't support real project to project references.

One workaround is to include all js files in the webproject by adding an item element similar to:
`<Content Include="output\ts\*.js />`
